code/__DEFINES/loadout.dm

```diff
@@ -0,0 +1,35 @@
+//List of all visible and accessible slot on the loadout maker
+GLOBAL_LIST_INIT(visible_item_slot_list, list(
+	slot_head_str,
+	slot_back_str,
+	slot_wear_mask_str,
+	slot_glasses_str,
+	slot_w_uniform_str,
+	slot_wear_suit_str,
+	slot_gloves_str,
+	slot_shoes_str,
+	slot_s_store_str,
+	slot_belt_str,
+	slot_l_store_str,
+	slot_r_store_str,
+))
+
+///All the vendor types which the automated loadout vendor can take items from
+GLOBAL_LIST_INIT(loadout_linked_vendor, list(
+	/obj/machinery/vending/marine/shared,
+	/obj/machinery/vending/uniform_supply,
+	/obj/machinery/vending/armor_supply,
+	/obj/machinery/vending/marineFood,
+))
+
+#define MARINE_LOADOUT "marine"
+#define ENGIE_LOADOUT "engie"
```

Comment:
Dont use an abbreviation use ENGINEER_LOADOUT to avoid confusion and make it easier to search